### PR TITLE
Feature/associate expenselog to logcategory

### LINKED ISSUE #115: Associate ExpenseLog to LogCategory
## Expected Behavior
An ExpenseLog should be associated with the logCategory, this will add any new expenseLog's created by the user. 

## Actual Behavior
The ExpenseLog is created with no association to the LogCategory

## Acceptance Criteria
The LogCategory should be associated to the ExpenseLog when created
## Steps to Reproduce the Problem

  1. Create an expense log using the expense log changeset

## Specifications

  - Version:

### PR DESCRIPTION
**Pull request by WanjikuMac**
Fixes #115 

## Proposed Changes

  -Associate the ExpenseLog to the LogCategory
  -
  -

### CODE REVIEW

File: lib/spender/using/expense_log.ex [hunk 1]
```diff
@@ -34,4 +37,16 @@ defmodule Spender.Using.ExpenseLog do
    |> cast(attrs, [:name, :desc, :amount])
    |> validate_required([:name, :amount])
  end
+ @spec create_changeset(Budget.t(), map) :: Changeset.t()
```
Comment (zacck-zz) [inline lib/spender/using/expense_log.ex:40]: Would you mind adding doc attributes to these functions?

File: priv/repo/migrations/20180731180224_associate_expenselog_to_logcategory.exs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+defmodule Spender.Repo.Migrations.AssociateExpenselogToLogcategory do
```
Comment (zacck-zz) [inline priv/repo/migrations/20180731180224_associate_expenselog_to_logcategory.exs:1]: Would you mind adding a module doc and doc attributes here too

**Review by zacck-zz — CHANGES_REQUESTED** [review]
Hey @WanjikuMac  this is awesome work! just a few changes to push it over the hill...

**Review by zacck-zz — APPROVED** [review]
Dope 🤙🏾

**Review by zacck-zz — APPROVED** [review]
Dope 🤙🏾

**Review by r-coh — APPROVED** [review]
Awsome.

**Pull request merged**